components/03-modules/header/header.hbs

```diff
@@ -1,78 +1,48 @@
                     >
-                        <svg class="header__icon">
-                            <use xlink:href="{{ svg item.icon }}"></use>
-                        </svg>
+                        {{ render '@icon' item.icon }}
+                        <span class="header-button__text">
+                            {{ item.text }}
+                        </span>
                     </a>
                 {{/each}}
                 <div class="header__minicart">
-                    {{ render '@minicart-trigger' trigger }}
+                    <a class="header-button popup-trigger"
+                       href="{{ minicartTrigger.link }}"
+                       aria-label="{{ minicartTrigger.ariaLabel }}"
+                       data-popuptrigger="popup-1"
+                    >
+                        {{ render '@icon' minicartTrigger.icon }}
+                        <span class="header-button__text">
+                            {{ minicartTrigger.text }}
+                        </span>
+                        <span class="header-button__counter">
+                            <span class="header-button__counter-dot"></span>
+                        </span>
+                    </a>
                     {{ render '@popup' minicartPopup }}
                 </div>
             </div>
-            <div class="header__menu-trigger">
-                {{ render '@side-menu-trigger--span-way' }}
-            </div>
-            <div class="header__featured-info">
-                <a href="#" class="featured-info">
-                    <div class="featured-info__icon">
-                        <svg class="featured-info__icon-svg">
-                            <use xlink:href="{{ svg 'curved-arrows' }}"></use>
-                        </svg>
-                    </div>
-                    <div class="featured-info__data">
-                        <div class="featured-info__data-title">
-                            30 days for return
-                        </div>
-                        <div class="featured-info__data-subtitle">
-                            see details
-                        </div>
-                    </div>
-                </a>
-                <a href="#" class="featured-info">
-                    <div class="featured-info__icon">
-                        <svg class="featured-info__icon-svg featured-info__icon-svg--delivery-truck">
-                            <use xlink:href="{{ svg 'delivery-truck' }}"></use>
-                        </svg>
-                    </div>
-                    <div class="featured-info__data">
-                        <div class="featured-info__data-title">
-                            Shipping in 24h
-                        </div>
-                        <div class="featured-info__data-subtitle">
-                            see details
-                        </div>
-                    </div>
-                </a>
-            </div>
-
+            {{ render '@logo' logo }}
```

Comment:
Logo hasn't got an alt attribute and logo link hasn't get `aria-label` or at least `title' set... Could you improve this in components or should we set the issue?

Reply:
fixed.